blue_modules/notifications.js

```
@@ -27,6 +26,25 @@ export const checkNotificationPermissionStatus = async () => {
 };
 
+/**
+ * Invalidate the stored push token, removing it from local storage.
+ */
+
+export const invalidateToken = async () => {
+  try {
+    const token = await AsyncStorage.getItem(PUSH_TOKEN);
+    if (token) {
+      const parsedToken = JSON.parse(token);
+      if (parsedToken && parsedToken.token) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)